### PR TITLE
ISO8601 version 0.2.5.

### PR DESCRIPTION
**Pull request by sagotch**

### CODE REVIEW

File: packages/ISO8601/ISO8601.0.2.5/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+build-doc: [ make "doc" ]
+
+build: [ make "build" ]
+
+install: [ make "install" ]
+
+remove: [ "ocamlfind" "remove" "ISO8601" ]
+
+depends: [ "ocamlfind" ]
```
Comment (dsheets) [inline packages/ISO8601/ISO8601.0.2.5/opam]: `{build}`

**Comment by dsheets** [comment]
Besides the fact that this is actually for the package ISO8601 0.2.5, this PR LGTM.

**Comment by sagotch** [comment]
Oups, you're right. Fixed.

File: packages/ISO8601/ISO8601.0.2.5/opam (hunk 1 above)
Comment (sagotch) [inline packages/ISO8601/ISO8601.0.2.5/opam]: What does that mean?

Comment (dsheets) [inline packages/ISO8601/ISO8601.0.2.5/opam]: `ocamlfind` should be marked as a build dependency as it is not used for the regular function of the package and an update to `ocamlfind` will not change the functionality or correctness of the package. This line should read something like `depends: [ "ocamlfind" {build} ]`.

Comment (sagotch) [inline packages/ISO8601/ISO8601.0.2.5/opam]: Oh yes, of course...

**Pull request merged**